banzai/tic-tac-toe.rb

```diff
@@ -0,0 +1,108 @@
+# this gem injects a color method to the string when it is displayed by console
+require 'colorize' 
```

Comment:
Add comment requiring to install colorize gem
=begin
colorize gem required to play game
gem install colorize
=end

Reply:
Gracias, ahora lo arreglamos.